.github/workflows/get-charm-paths.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -x
```

Comment:
I like this for figuring the charm paths, but I think it should live in some shared repos. Maybe put it in charming actions?

Reply:
100% agreed.  Only question to me is schedule - do we have time to make it into an action, or should we push this first and then later replace with an action?  I'll poke @DomFleischmann

Reply:
Discussed with @DomFleischmann and we will push the script for now, but make this into a real action in the near term